src/components/MessageCommerce.js

```diff
@@ -220,27 +243,14 @@ class MessageCommerce extends PureComponent {
       message.latest_reactions && message.latest_reactions.length,
     );
 
-    if (
-      message.type === 'message.read' ||
-      message.deleted_at ||
-      message.type === 'message.date'
-    ) {
-      return null;
+    if (message.deleted_at) {
+      return <MessageDeleted {...this.props} />; // MessageDeleted is always defined because it has a default value
```

Comment:
For these renders you probably want to use this helper function:
https://github.com/GetStream/stream-chat-react/blob/985a7b235156942a51718769adc40e897f71ebc5/src/utils.js#L205

Reply:
That's useful, didn't know that existed in the utils.